bin/all_sky_search/pycbc_calculate_dqflag

```diff
@@ -26,6 +26,8 @@ parser.add_argument('--flag', type=str, required=True,
                     help="name of the data quality flag")
 parser.add_argument('--ifo', type=str, required=True,
                     help="interferometer to analyze")
+parser.add_argument('--sample-frequency', required=False,
+                    default=128, help="timeseries sampling frequency")
```

Comment:
Why 128 as default? Seems a rather large value compared to 1 as used before. Do you need a default anyway?

Also suggest adding units to the help ie `(Hz)`

Reply:
Many of the vetoes I want to use have durations as little as 1/4 of a second. The idq timeseries itself has a sample rate of 1024 so theoretically vetoes can start/end on any 1/1024th of a second. 128 felt like a reasonable setting to pick up on a flag starting quickly without actually checking every 1024th of a second. But I can easily make this configurable.